e2e/util/validation.go

```diff
@@ -115,3 +115,42 @@ func FindPod(cluster types.Cluster, namespace, labelSelector, podIdentifier stri
 	return nil, fmt.Errorf("no pod with label selector %q and identifier %q in namespace %q in cluster %q",
 		labelSelector, podIdentifier, namespace, cluster.Name)
 }
+
+// ValidateClustersInDRPolicy checks if c1 and c2 exists in the DRPolicy.
+// Returns an error if either cluster is missing.
+func ValidateClustersInDRPolicy(env *types.Env, config *types.Config, log *zap.SugaredLogger) error {
```

Comment:
We have lot of code accepting env, config, log - this is basically the the global context (compared to a test context). We need to think about a global context object that can be passed to all function not called only during a test.

For now we can go with what you added.